pages/concludingStudents.vue

```diff
@@ -1,11 +1,48 @@
 <template>
   <div class="container">
+    <div class="columns is-centered">
+      <div class="column is-5 is-offset-2">
+        <h1 class="title">
+          <strong>Alunos Concluintes</strong>
+        </h1>
+      </div>
+    </div>
+    <div class="columns">
+      <div class="column is-3 is-offset-8 to-left">
+        <b-button class="is-primary" @click="getAcademicHighlights">
+          Gerar lista de destaques acadêmicos</b-button
+        >
```

Comment:
Nope

Reply:
Se for a formatação da tag HTML, é culpa do prettier

Reply:
Seja mais específico. Evite dizer "Nope", nem sempre fica claro pra quem lê

Reply:
Só tinha um caracter na linha do nope. Não tinha muitas alternativas.

Da pra melhorar.